semgrep-core/src/runner/Run_semgrep.ml

```diff
@@ -743,6 +743,7 @@ let minirule_of_pattern lang pattern_string pattern =
     message = "";
     severity = R.Error;
     languages = [ lang ];
+    fix = None;
```

Comment:
It bothers me a little bit that we can't distinguish between "this rule has no fix" and "this rule has a failed fix". I can't think of any place where this could cause problems, it just makes me suspicious.

Reply:
This is the minirule field that has the rule's `fix` field as written (if it has one). At this point, we haven't attempted to render a fix for a particular finding.